### PR TITLE
Popup

### PR DESCRIPTION
**Pull request by AminaBuhari**
I added a popup window and added javascript  to the code. I also updated the index.html and the style.css.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+              <a href="${project.source}"><button class="button-mo1">
+                  See Source
+                  <img src="images/live.png" alt="git-hub icon" class="pop-icon" />
+              </button></a>
+          </div>
+    
+  </div>
+</section>`;
+  bodyElement.insertAdjacentHTML('afterbegin', popupHTML);
+}
```
Comment (gemmen29) [inline script.js:167]: Kindly try to match the Figma design for the modal in the mobile version. Please make the close button like the Figma design and make the modal scrollable so the full modal appears.
| Your design | Figma design |
| --- | --- | 
| image | image |

Comment (gemmen29) [inline script.js:167]: Kindly try to match the Figma design for the modal in the desktop version width 992px. Please make the header, list of technologies, and  `see live` and `see source` buttons the Figma design and make the modal scrollable so the full modal appears.
| Your design | Figma design |
| --- | --- | 
| image | image |

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @AminaBuhari, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AminaBuhari** [comment]
Thank you for the review. I made some changes to the style and would like to know your recommendation
Screenshot 2022-03-04 at 11-56-49 Amina Portfolio Project
Screenshot 2022-03-04 at 11-56-18 Amina Portfolio Project
.

**Review by gemmen29 — APPROVED** [review]
Hi @AminaBuhari, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions:
- Kindly add the close button for the modal in mobile version to match the Figma design

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**